packetbeat/etc/beat.yml

```diff
@@ -17,6 +17,10 @@ interfaces:
 
 ############################# Protocols #######################################
 protocols:
+  icmp:
+    # Enable ICMPv4 and ICMPv6 monitoring. Default: false
+    enabled: true
```

Comment:
The defaults in Packetbeat are that all protocol plugins are disabled. So it is normal for Packetbeat to write `enabled: true` even though the default is false. It's similar to writing `ports: ['80', '8080']` to enable `http` in the config file.